File: java/test/ExistsTest.java
package voxgig.bluefintecsecrsdk.sdktest;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;

import voxgig.bluefintecsecrsdk.core.BluefinTecsEcrSDK;

public class ExistsTest {

  @Test
  public void testMode() {
    BluefinTecsEcrSDK testsdk = BluefinTecsEcrSDK.testSDK();
    assertNotNull(testsdk, "expected non-nil SDK");
  }
}

File: kotlin/test/ExistsTest.kt
package voxgig.bluefintecsecrsdk.sdktest

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

import voxgig.bluefintecsecrsdk.core.BluefinTecsEcrSDK

class ExistsTest {

  @Test
  fun testMode() {
    val testsdk = BluefinTecsEcrSDK.testSDK()
    assertNotNull(testsdk, "expected non-nil SDK")
  }
}
